src/constants.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+module.exports.DEFAULT_COMMAND = `--help`;
+
+module.exports.USER_ARGV_INDEX = 2;
+
+module.exports.exitCode = {
```

Comment:
```suggestion
module.exports.ExitCode = {
```
енамы пишутся с большой буквы